Documentation/ProjectFile/prj/processes/process/TWOPHASE_FLOW_PP/material_property/fluid/i_fluid.md

```diff
@@ -0,0 +1 @@
+A tag for fluid properties of gas and liquid phases.
```

Comment:
I will update those for LiquidFlow according to the changes will be done in this PR.